src/buildHTML.js

```diff
@@ -750,8 +802,8 @@ groupTypes.op = function(group, options, prev) {
         // If this is a symbol, create the symbol.
         var fontName = large ? "Size2-Regular" : "Size1-Regular";
         base = buildCommon.makeSymbol(
-            group.value.body, fontName, "math", options.getColor(),
-            ["op-symbol", large ? "large-op" : "small-op", "mop"]);
+            group.value.body, fontName, "math", options,
+            ["mop", "op-symbol", large ? "large-op" : "small-op"]);
```

Comment:
Thanks... this order for the classes is better.  It would be nice if the param order between `makeSpan` and `makeSymbol` was more similar.  Can you add a TODO to clean that up later (there's already enough changes in this PR)?